platform/broadcom/sonic-platform-modules-brcm-xlr-gts/Makefile

```diff
@@ -7,7 +7,7 @@ MAIN_TARGET = $(BRCM_XLR_GTS_PLATFORM_MODULE)
 $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	# Build the package
 	export PYBUILD_INSTALL_ARGS_python2=--install-scripts=/dev/null
-	dpkg-buildpackage -rfakeroot -b -us -uc -j$(SONIC_CONFIG_MAKE_JOBS)
+	dpkg-buildpackage -rfakeroot -b -us -uc -j$(SONIC_CONFIG_MAKE_JOBS) --admindir $(SONIC_DPKG_ADMINDIR)
```

Comment:
>--admindir $(SONIC_DPKG_ADMINDIR) [](start = 69, length = 33)

In stead of modifying every command line, you may modify global config file `/usr/local/etc/dpkg/buildpackage.conf`.
So there is no risk to miss it.

Reply:
I cannot.  the admindir is unqiue per package build.